epochStart/metachain/economics.go

```diff
@@ -186,11 +188,12 @@ func (e *economics) adjustRewardsPerBlockWithLeaderPercentage(
 	rwdPerBlock.Sub(rwdPerBlock, averageLeaderRewardPerBlock)
 }
 
-// compute inflation rate from totalSupply and totalStaked
-func (e *economics) computeInflationRate(_ *big.Int, _ *big.Int) (float64, error) {
-	//TODO: use prevTotalSupply and nodePrice (number of eligible + number of waiting)
-	// for epoch which ends now to compute inflation rate according to formula provided by L.
-	return e.rewardsHandler.MaxInflationRate(), nil
+// compute inflation rate from genesisTotalSupply and economics settings for that year
+func (e *economics) computeInflationRate(currentRound uint64) float64 {
+	roundsPerDay := numberOfSecondsInDay / uint64(e.roundTime.TimeDuration().Seconds())
+	roundsPerYear := numberOfDaysInYear * roundsPerDay
+	yearsPassed := uint32(currentRound/roundsPerYear) + 1
```

Comment:
The name is misleading. Might change to yearIndex as for round 0 this parameter will hold the value of 1 and is called yearsPassed.

Reply:
done